.github/workflows/update_prices.yaml

```diff
@@ -16,7 +16,8 @@ jobs:
 
       - name: Build publisher images locally
         run: |
-          docker build . -t 42labs/pontis-publisher:latest
+          export PONTIS_PACKAGE_VERSION=$(grep 'version' pontis-package/setup.cfg | grep -e '[0-9]\.[0-9]\.[0-9]' -o)
```

Comment:
If I'm understanding this correctly, new line 19 and 20 is semantically equivalent to old line 19, except you use the grep on new line 19 to make sure you have the right Pontis package version from the setup config, and from there you rebuild as you would on old L19, except by specifying the latest package version as an argument. 

Nice this reminds me of the monitor setup shell script fixes, except while there we limited automatic building with latest, we encourage it here. Is that correct?